packages/docusaurus-theme-classic/src/theme/LastUpdated/index.tsx

```diff
@@ -59,7 +57,7 @@ export default function LastUpdated({
   lastUpdatedBy,
 }: Props): JSX.Element {
   return (
-    <div className={clsx('col', styles.lastUpdated)}>
```

Comment:
Like the `EditThisPage` component, `LastUpdated` component should _not_ use the grid system (`col` class), but simply display the content it needs.

Reply:
agree